common/containers/Tabs/Contracts/components/Deploy/components/DeployHoc/components/TxCompare.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Wei } from 'libs/units';
+import translate from 'translations';
+export interface Props {
+  nonce: string;
+  gasPrice: Wei;
+  gasLimit: string;
+  to: string;
+  value: string;
+  data: string;
+  chainId: number;
+  signedTx: string;
+}
+
+export const TxCompare = (props: Props) => {
+  const { signedTx, ...rawTx } = props;
+  const Left = () => (
+    <div className="form-group">
+      <h4>{translate('SEND_raw')}</h4>
+      <textarea
```

Comment:
These `<textarea/>`'s are pretty tiny, could use with some changes to the css so they are more readable